db/migrate/20161212003047_add_payments.rb

```diff
@@ -0,0 +1,10 @@
+# coding: utf-8
+# frozen_string_literal: true
+class AddPayments < ActiveRecord::Migration
+  def change
+    create_table :payments do |t|
+      t.integer :amount, null: false
```

Comment:
that was very interesting to know, I agree that using integers for calculation than  dividing by 100 for display is the best choice. I had done the mistake of using decimal type for currency in an old project :cry: